recipes/ensembl-vep/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   md5: ad9059806b061ac11bb3e7d9f51ca30d
 
 build:
-  number: 0
+  number: 1
   skip: True # [osx]
```

Comment:
@chapmanb if such a recipe has a build-string containing CONDA_HTSLIB you don't need to increase the build number, in fact it should be resetted.
A few recipes do not use the build-string ... this seems to be wrong :(